src/config/paths.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Paths
+ */
+
+export enum paths {
```

Comment:
Oops, wrong one, enums are PascalCase ;-)
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#enum

I think this file was good as it was before, also regarding its name (Path.ts) and the enum values (for which we decided to keep upper case)